### DISCUSSION TITLE
OpenSSL version?

### DISCUSSION THREAD

**Discussion by mig5** [ory/hydra#3321] [Q&A] [answered]
Hi,

Can you advise if the Hydra v1.11.10 container links against/uses/compiles OpenSSL as part of its functionality, and if so, which version? I'm not too good with Golang and struggling to find out for myself in the source code..

Thanks!

**Comment by aeneasr** [comment, marked as answer]
Go does not use OpenSSL, it uses https://pkg.go.dev/crypto/tls

**Reply by mig5** [reply]
Thank you for the quick answer!
